Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -342,26 +342,35 @@ private void OnDefaultGUI()
                     keyLength + ".", MessageType.Warning);
             }
 
-            GUIEditorUtility.AssigningFlagTextField("Platform Flags (Seperated by '|')",
-                ref mainEOSConfigFile.platformOptionsFlags, 190,
-                "Flags used to initialize EOS Platform. Available flags are defined in PlatformFlags.cs");
-            GUIEditorUtility.AssigningFlagTextField("Auth Scope Flags (Seperated by '|')",
-                ref mainEOSConfigFile.authScopeOptionsFlags, 210,
-                "Flags used to specify Auth Scope during login. Available flags are defined in AuthScopeFlags.cs");
+#if !EOS_DISABLE
+            GUIEditorUtility.AssigningEnumField(
+                "Platform Flags",
+                ref mainEOSConfigFile.platformOptionsFlags,
+                190,
+                "Flags used to initialize EOS Platform. Available " +
+                "flags are defined in PlatformFlags.cs");
+
+            GUIEditorUtility.AssigningEnumField(
+                "Auth Scope Flags",
+                ref mainEOSConfigFile.authScopeOptionsFlags,
+                210,
```

Comment:
Can this be changed to 190? It'd line things up better, I think.
![image](https://github.com/user-attachments/assets/e7195b3a-25e6-4657-a9e7-6c24573fc4c3)

Reply:
Good idea - fixed.